spec/models/item_spec.rb

```diff
@@ -43,4 +39,27 @@
   xit 'responds to orders' do
     expect(item).to respond_to(:orders)
   end
+
+  it 'must have a unique title' do
+    item
+    item2 = Item.new(title: "Green Chili Burrito", description:"Amazingly spicey burrito", price:655, image: "image/path")
+    expect(item2).to_not be_valid
+  end
+
+  xit 'must belong to at least one category' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.